ams/optimizer/common/src/main/java/com/netease/arctic/optimizer/common/AbstractOptimizerOperator.java

```diff
@@ -16,6 +16,8 @@
 
 public class AbstractOptimizerOperator implements Serializable {
   private static final Logger LOG = LoggerFactory.getLogger(AbstractOptimizerOperator.class);
+
+  @SuppressWarnings("all")
```

Comment:
Maybe add a final modifier rather than the `suppressWarnings` annotation?

Reply:
> Maybe add a final modifier rather than the `suppressWarnings` annotation?

Because of unit test helper use reflection to change this value.

Reply:
> How about expose overwrite / set method for this field and without final be applied to this field?

Yes, this is indeed a better way.